### PR TITLE
Include completion information in controller's tasks

### PR DESCRIPTION
**Pull request by shrutiparabgoogle**
This PR includes the following changes:
1) Setting the output of child commands to StdOut and StdErr so that the logs generated by the child commands are all visible.
2) `local-tasks.go`: Each individual go routine will log the completion of a command execution by logging the status, action and taskID (will be tied to child command's uid in the future). This information should be enough for now to generate basic summary dashboards like: total num actions, num failures, % success, etc

### CODE REVIEW

File: cmd/registry/controller/local-tasks.go [hunk 1]
```diff
@@ -36,12 +37,14 @@ func (task *ExecCommandTask) Run(ctx context.Context) error {
 		return fmt.Errorf("'resolve' not allowed in action, cannot invoke %q", task.Action)
 	}
 	cmd := exec.Command("registry", strings.Fields(task.Action)...)
-	output, err := cmd.CombinedOutput()
+	cmd.Stdout, cmd.Stderr = os.Stdout, os.Stderr
+	err := cmd.Run()
+	details := fmt.Sprintf("Finished execution %%s action={%s} taskID={%s}", task.Action, task.TaskID)
```
Comment (shrutiparabgoogle) [inline cmd/registry/controller/local-tasks.go:42]: taskID can later be passed as a flag to the child process as it's uid.

**Comment by shrutiparabgoogle** [comment]
Planning to build a basic dashboard based on the uid prefixes and this information. Will then look for enhancements, once we have a connection between the parent generated taskID and the child's uid.

File: cmd/registry/controller/local-tasks.go [hunk 2]
```diff
@@ -36,12 +37,14 @@ func (task *ExecCommandTask) Run(ctx context.Context) error {
...
 
 	if err != nil {
+		log.Printf(details, "FAILED")
 		return err
 	}
-	log.Printf("Finished executing taskId %s with action: %s\n Output: %s", task.TaskID, task.Action, output)
-
+	log.Printf(details, "SUCCESS")
```
Comment (seaneganx) [inline cmd/registry/controller/local-tasks.go:48]: Using the escaped formatter from the details variable is a bit confusing here. Could we capture less information in the details variable? These log messages can be `log.Printf("Failed execution of %s: %s", details, err)` and `log.Printf("Successful execution of %s", details)` if `details` only include the action and task ID.

Alternatively I actually think it's clearer to simply have two fully separate log messages that just repeat the common details, but there's no right answer for that.

**Review by seaneganx — APPROVED** [review]

File: cmd/registry/controller/local-tasks.go (hunk 2 above)
Comment (shrutiparabgoogle) [reply cmd/registry/controller/local-tasks.go]: Right, thanks for calling it out. I ended up over optimizing the code. Made changes, hope it's more clear now

**Pull request merged**